### ISSUE TITLE
pretrained model 

### ISSUE THREAD

**Issue by nitba** [octiapp/KerasPersonLab#10]
Hi @jricheimer 

Thank you so much for your implementation, It would be great if we can get access to pretrained model to compare it's accuracy with paper.

Thanks,

**Comment by AroMorin** [comment]
That would be helpful indeed, any luck?
